Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -2477,13 +2477,20 @@ static inline bool rectIsTooBigForSelection(const IntRect& blockRect, const Loca
     constexpr OptionSet hitType { HitTestRequest::Type::ReadOnly, HitTestRequest::Type::Active, HitTestRequest::Type::AllowVisibleChildFrameContentOnly };
     auto* localMainFrame = dynamicDowncast<LocalFrame>(m_page->mainFrame());
     if (!localMainFrame)
-        return;
+        return completionHandler(false, { });
     Ref frame = *localMainFrame;
     auto result = frame->eventHandler().hitTestResultAtPoint(point, hitType);
     RefPtr hitNode = result.innerNonSharedNode();
     if (!hitNode)
         return completionHandler(false, { });
 
+    if (auto* view = frame->view()) {
+        LayoutPoint vPoint = view->windowToContents(point);
```

Comment:
Hmm... is `windowToContents` the right transformation here? The `point` comes directly from the UI process via UIKit code that works with `WKContentView`, so I believe it's in root view coordinates, in which case `view->rootViewToContents(point)` should be preferable.

That said, it's possible that since `frame` this is only ever the main frame here, it doesn't actually make a difference.

Nit - instead of `vPoint`, we should write out [full words unless it's a well-known abbreviation](https://webkit.org/code-style-guidelines/#names-full-words), and rename it to something like `viewPoint` or (perhaps more accurate) `pointInContents`.

Reply:
Can you tell I just copy-pasted this line 😛 Will fix, thanks!